modules/tinymce/src/core/main/ts/newline/InsertLi.ts

```diff
@@ -48,11 +50,12 @@ const isFirstOrLastLi = (containerBlock, parentBlock, first) => {
 };
 
 // Inserts a block or br before/after or in the middle of a split list of the LI is empty
-const insert = (editor: Editor, createNewBlock, containerBlock, parentBlock, newBlockName: string) => {
+const insert = (editor: Editor, createNewBlock: (name: string) => Element, containerBlock: Element, parentBlock: Element, newBlockName: string): void => {
   const dom = editor.dom;
   const rng = editor.selection.getRng();
+  const containerParent = containerBlock.parentNode;
 
-  if (containerBlock === editor.getBody()) {
+  if (containerBlock === editor.getBody() || !containerParent) {
```

Comment:
Code change here to early exit if we don't have a parent node for the container (i.e. the container is detached).